server/container_status.go

```diff
@@ -101,6 +102,11 @@ func (s *Server) ContainerStatus(ctx context.Context, req *pb.ContainerStatusReq
 	resp.Status.State = rStatus
 	resp.Status.LogPath = c.LogPath()
 
+	resp.Info = map[string]string{
```

Comment:
We should check if `Verbose` is `true` in the request before adding this as per:
```
 2461     // It should only be returned non-empty when Verbose is true.
```

Reply:
Fixed.